querqy-core/src/main/java/querqy/rewrite/RewriterFactory.java

```diff
@@ -31,7 +31,7 @@ public abstract QueryRewriter createRewriter(ExpandedQuery input,
      *
      * @return The set of {@link Term}s to be generated.
      */
-    public abstract Set<Term> getGenerableTerms();
+    public abstract Set<Term> getCacheableGenerableTerms();
 
```

Comment:
Minor thing: As there are a number of rewriter implementations out there, can we leave `getGenerableTerms()` in, deprecate it, and implement it so that it delegates to `getCacheableGenerableTerms()`?

Reply:
Implemented in 283b012